dev/format_code.sh

```diff
@@ -1,14 +1,35 @@
 #!/usr/bin/env bash
+set -eo pipefail
 
 # Formats all files which differ from their state on the master branch with
 # clang-format.
 #
 # Usage:
 #   - Make sure the executables "clang-format" and "git" are available in PATH.
 #   - Run the command "./dev/format_code.sh" in the root of the repository.
+#   - To run clang-format in a docker-container, use the "--docker" parameter.
 
 echo "Formatting modified files with clang-format..."
 
+DOCKER_IMAGE=librepcb/clang-format:6
+if [ "$1" == "--docker" ]; then
+  if [ "$(docker images -q $DOCKER_IMAGE | wc -l)" == "0" ]; then
+    echo "Building clang-format container..."
+    cd clang-format
```

Comment:
This fails when running the script from the repository's root directory (according usage instructions).

Reply:
Oops, I used repo root detection when running, but not when building :)